app/platform_api/schema/invoice_summary.rb

```diff
@@ -110,7 +110,7 @@ def self.id_ref
   # The outstanding balance amount in the group's base currency.
   # @type: number
   # @format: double
-  field :base_currency_outstanding_balance_amount
```

Comment:
should this be changed for invoice model as well?

Reply:
NO invoice model we are getting the response from v1/Invoices/query there we have the column name this only - "baseCurrencyOutstandingBalanceAmount" @rishirajkumar97

Reply:
hmm is platform fields are varying between Summary endpoints vs the actual invoice endpoint?

Reply:
Yes